backend/pkg/service/service_test.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2019 The OpenSDS Authors.
```

Comment:
Please change it to SODA Authors
```suggestion
// Copyright 2019 The SODA Authors.
```